### DISCUSSION TITLE
Add light/dark theme switch to menu?

### DISCUSSION THREAD

**Discussion by lfenzo** [imfing/hextra#341] [Q&A] [answered]
Is there any way to move the light/dark theme switch to the menu at the top of the page?

**Comment by imfing** [comment, marked as answer]
The feature you're asking about isn't directly supported by default. 
However, you can achieve this by customizing the `navbar.html` partial located at [`layouts/partials/navbar.html`](https://github.com/imfing/hextra/blob/main/layouts/partials/navbar.html) in the project repository. To add a theme toggle, you would manually insert the `theme-toggle` component into this file.

For a practical example of how this can be customized, you might want to check out [this website](https://infracourse.cloud/) which demonstrates the functionality in action.

I'll open an issue from this and work on supporting this in the future.

**Reply by lfenzo** [reply]
Thanks @imfing!

**Comment by dogweather** [comment]
This is on my list to do as well. I'll give it a shot too.
